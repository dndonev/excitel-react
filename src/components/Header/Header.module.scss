@import '../../variables';

.Header {
    width: 100%;
    display: flex;
    flex-direction: row;
    gap: $spacings-base;  
    justify-content: center;
    align-items: center;

    padding: $spacings-base;
    box-sizing: border-box;
 
    text-transform: uppercase;

    background-color: $background-white;
    box-shadow: 3px 3px 5px 6px #ccc;

    span {
        font-weight: 700;
        letter-spacing: $spacings-base;
        color: #2096F3;
    }
}