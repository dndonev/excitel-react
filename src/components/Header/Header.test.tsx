import Header from './Header';

describe('<Header />', () => {
  // it('should render correctly with empty props', () => {

  //   const component = shallow(<Header />);
    
  //   expect(component).toMatchSnapshot();
  // });
});