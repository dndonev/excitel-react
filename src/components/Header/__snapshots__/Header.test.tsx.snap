// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly with empty props 1`] = `ShallowWrapper {}`;
